test/test_jit.py

```diff
@@ -5215,6 +5215,51 @@ def test_pow_int(x, y):
             # type: (float, int) -> float
             return math.pow(x, y)
 
+        def test_isnan(x):    #HERE
```

Comment:
Potential refactoring: 
If you look at our test_number_math, we have one template that we substitute in different functions into.
It would be nice to do something similar here. By having the scalar literal in the actual function body, you don't have to add all of the type comments, which simplifies things.

Reply:
I think this is a good idea, but maybe it would be better to wait for the others working on other functions to wrap up and then refactored it since this would affect their work?

Reply:
Thoughts @eellison as people are working on the other functions?

Reply:
I think it's still worth it to refactor

Reply:
I'll start working on this.  Will this then deprecate the existing tests

Reply:
Updated, take a look